pkg/cloudclient/gcp/gcp.go

```diff
@@ -28,15 +28,22 @@ var (
 	log = logf.Log.WithName("gcp_cloudclient")
 )
 
+type GCPComputeClient interface {
+	GetForwardingRuleList() (*computev1.ForwardingRuleList, error)
+}
+
 // Client represents a GCP cloud Client
+// To recreate mock for this, run:
+// mockgen -source=pkg/cloudclient/gcp/gcp.go -destination=pkg/cloudclient/mocks/mock_gcp.go
```

Comment:
This is a totally different repo, so not necessarily a good idea, but should we add a Makefile target to generate these mocks like: https://github.com/openshift/osd-network-verifier/blob/main/Makefile#L24-L28 ?

Reply:
I like that! I'll try it out. Also, just realized the path to destination isn't right..